cosmos/scheduler/cmd/scheduler.go

```diff
@@ -149,8 +150,7 @@ func (s *Scheduler) watchOverlayUpdates() (chan interface{}, error) {
 
 func (s *Scheduler) getIngestionSecret() (string, error) {
 	secrets, err := s.KubeClientset.CoreV1().
-		Secrets(s.Namespace).
-		List(metav1.ListOptions{LabelSelector: "ingestionCredentials=true"})
+		Secrets(s.Namespace).Name(s.SecretName)
```

Comment:
is `.Name` a method? can't find the definition for it (looking at: https://github.com/kubernetes/client-go/blob/acc621f88da0907c3eee966001ea99bd59b0f92c/kubernetes/typed/core/v1/secret.go#L39)

Reply:
Yeah I just realized this as well